pysat/tests/test_meta.py

```diff
@@ -67,6 +70,43 @@ def test_default_label_value_raises_error(self):
 
         assert verr.match("unknown label attribute")
 
+    def test_default_value_from_type_unexpected_input(self, caplog):
+        """ Test MetaLabels.default_values_from_type with unexpected input
+        """
+        unexpected = 1
+        with caplog.at_level(logging.INFO, logger='pysat'):
+            self.meta.labels.default_values_from_type(unexpected)
+
+            captured = caplog.text
+
+            # Test for expected string
+            test_str = 'No type match found for '
+            assert captured.find(test_str) >= 0
+
+        return
+
+    @pytest.mark.parametrize("input",
+                             [float, np.float64, np.float32])
```

Comment:
```suggestion
                             [float, np.float16, np.float32, np.float64, np.float128])
```

Reply:
GitHub Actions doesn't support np.float128 but I've taken the rest of your suggestion.